suites/quincy/nvmeof/ceph_nvmeof_baremetal_scale.yaml

```diff
@@ -0,0 +1,282 @@
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  #  Test cases to be executed
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node5
+          - node6
+          - node7
+          - node8
+          - node9
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client
+      polarion-id:
+        CEPH-83573758
+
+  - test:
+    abort-on-fail: true
+    config:
+      gw_node: node5
+      rbd_pool: rbd
+      do_not_create_image: true
+      rep-pool-only: true
+      rep_pool_config:
+        pool: rbd
+      install: true
+      subsystems:
+        - nqn: nqn.2016-06.io.spdk:cnode1
+          serial: 1
+          max_ns: 64
+          bdevs:
+            count: 64
+            size: 50G
+          listener_port: 5001
+          allow_host: "*"
+      initiators:
+        - subnqn: nqn.2016-06.io.spdk:cnode1
+          listener_port: 5001
+          node: node6
+      run_io:
+        - node: node6
+          size: 20%
+          io_type: write
+    desc: test with 1 subsystem and 64 namespaces in Single GW
+    destroy-cluster: false
+    module: test_ceph_nvmeof_gateway_scale.py
+    name: Scale subsystems and namespaces in Single NVMeOF GW
+    polarion-id:
+
+  - test:
+      name: check-ceph-image-usage
+      module: exec.py
+      config:
+        cmd: rbd du
+        sudo: true
+      desc: Check for rbd image usage info
+
+  - test:
+      name: check-ceph-health
+      module: exec.py
+      config:
+        cmd: ceph -s
+        sudo: true
+      desc: Check for ceph health debug info
+
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node5
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        subsystems:
+          - nqn: nqn.2016-06.io.spdk:cnode2
+            serial: 2
+            max_ns: 64
+            bdevs:
+              count: 64
+              size: 50G
+            listener_port: 5002
+            allow_host: "*"
+        initiators:
+          - subnqn: nqn.2016-06.io.spdk:cnode2
+            listener_port: 5002
+            node: node7
+        run_io:
+          - node: node6
+            size: 5%
+            io_type: read
+          - node: node7
+            size: 20%
+            io_type: write
+      desc: test with 2 subsystems and 128 namespaces in Single GW
+      destroy-cluster: false
+      module: test_ceph_nvmeof_gateway_scale.py
+      name: Scale subsystems and namespaces in Single NVMeOF GW
+      polarion-id:
+
+  - test:
```

Comment:
There are multiple occurrences of the same test cases for checking Ceph image usage and health. Can we consolidate and have them in the same testss instead of caling them seapartely each tim. We can inlcude repeated test cases into a single instance

Reply:
Thanks for the review. 
I think we cannot consolidate these checks to main tests as they use separate test modules. But can you please share me a link or example how we can achieve this?

Reply:
Ceph health check and ceph usage are part of the test case as validation check points. We should have the code flow in the same module. We can define a method and call the function in the same module.

Reply:
addressed this